JSDemos/Demos/DataGrid/PDFOverview/description.md

```diff
@@ -1,8 +1,12 @@
-Our [DataGrid](/Documentation/ApiReference/UI_Components/dxDataGrid/) allows you to easily and accurately export its contents to a PDF document. To enable PDF export operations, you must reference or import the following:
+The [DataGrid](/Documentation/ApiReference/UI_Components/dxDataGrid/) allows you to export its contents to a PDF document. To enable PDF export operations, import the <a href="https://github.com/parallax/jsPDF" target="_blank">jsPDF</a> library.
 
-- <a href="https://github.com/parallax/jsPDF" target="_blank">jsPDF</a>        
-A library that creates and manages PDF documents.
+Call the [exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/pdfExporter/#exportDataGridoptions) method that belongs to the [pdfExporter](/Documentation/ApiReference/Common/Utils/pdfExporter/) module to export DataGrid content to a PDF document. Specify at least two required properties:
```

Comment:
I don't want to repeat the part that says "to export DataGrid content to a PDF document" because the first sentence in this topic already says that and uses almost all the same words. So this reads repetitive. And that phrase complicates the sentences (long sentences that has too many words "to").

```suggestion
Call the [exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/pdfExporter/#exportDataGridoptions) method that belongs to the [pdfExporter](/Documentation/ApiReference/Common/Utils/pdfExporter/) module. Specify at least two required properties:
```